optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/employee/EmployeeDataFetchable.java

```diff
@@ -35,88 +35,28 @@
     EmployeeRosterView last;
     Calendar<EmployeeId, EmployeeData> calendar;
     static final String LOADING_STRING = "Fetching Employee Roster";
```

Comment:
i18n

Reply:
Actually not i18n; it is a unique identifier String used by LoadingPopup; it isn't display anywhere; probably can be moved to an enum.